src/main/java/click/escuela/grade/model/Grade.java

```diff
@@ -0,0 +1,50 @@
+package click.escuela.grade.model;
+
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "grade")
+@Entity
+@Builder
+public class Grade {
+	@Id
+	@Column(name = "id", columnDefinition = "BINARY(16)")
+	@GeneratedValue(generator = "uuid2")
+	@GenericGenerator(name = "uuid2", strategy = "uuid2")
+	private UUID id;
+
+	@Column(name = "name", nullable = false)
+	private String name;
+
+	@Column(name = "course", nullable = false)
+	private String course;
+	
+	@Column(name = "subject", nullable = false)
+	private String subject;
+	
+	@Column(name = "type", nullable = false)
+	private String type;
```

Comment:
In the field type Can we change String to Enum? If this is very difficult we can add it in another story

Reply:
But what kind of "type" do youn want? Give me some examples so I can put in TypeEnum.